Dockerfile

```diff
@@ -330,4 +331,5 @@ USER tgis
 EXPOSE ${PORT}
 EXPOSE ${GRPC_PORT}
 
+ENTRYPOINT ["wait-modelcar.sh"]
```

Comment:
It might be more streamlined to only have a single `COMMAND` with two arguments.